py-bindings/pyproject.toml

```diff
@@ -1,13 +1,31 @@
+[project]
+name = "ompl"
+version = "1.7.0"
+description = "The Open Motion Planning Library"
+authors=[
+    {name = "Ioan A. Șucan", email = "ioan@sucan.ro"},
+    {name = "Mark Moll", email = "mark@moll.ai"},
+    {name = "Zachary Kingston", email = "zkingston@purdue.edu"},
+    {name = "Lydia E. Kavraki", email = "kavraki@rice.edu"}
+]
```

Comment:
Consider adding something like:
```
classifiers = [
  "Topic :: Scientific/Engineering"
]
```

There is a full list of classifiers somewhere.